src/js/app.js

```
@@ -854,17 +863,22 @@ globalThis.clearAllLanguages = clearAllLanguages;
-  else{chips.add(k);el.className='chip '+chipCls[k]}
+  if(chips.has(k)){chips.delete(k);el.className='chip';}
+  else{chips.add(k);el.className='chip '+chipCls[k];}
   applyFilters();
 }
-function resetFilters(){
-  ['searchInput','catFilter','langFilter','yearsFilter','compFilter'].forEach(id=>{const e=document.getElementById(id);if(e)e.value=''});
+function _resetFilters(){
+  ['searchInput','catFilter','langFilter','yearsFilter','compFilter'].forEach(id=>{const e=document.getElementById(id);if(e)e.value='';});
   document.getElementById('sortSelect').value='alpha';
   pills.clear();chips.clear();
+HEAD
```

ESLint

[error] 'HEAD' is not defined.

(no-undef)